server/service/hosts.go

```diff
@@ -834,6 +835,19 @@ func (svc *Service) getHostDetails(ctx context.Context, host *fleet.Host, opts f
 	}
 	host.MDM.Profiles = &profiles
 
+	var macOSSetup *fleet.HostMDMMacOSSetup
+	if ac.MDM.EnabledAndConfigured {
+		macOSSetup, err = svc.ds.GetHostMDMMacOSSetup(ctx, host.ID)
+		if err != nil {
+			if !fleet.IsNotFound(err) {
+				return nil, ctxerr.Wrap(ctx, err, "get host mdm macos setup")
+			}
+			// TODO(Sarah): What should we do for not found? Should we return an empty struct or nil?
```

Comment:
Good question. I think in the past we've returned empty struct but you may want to validate with Luke when he's back.